src/style.css

```diff
@@ -0,0 +1,150 @@
+body {
+  background-color: #efefef;
+}
+
+.list {
+  box-shadow: 2px 2px 2px 2px rgb(0 0 0 / 10%);
+  background-color: #fff;
+  margin: auto;
+  margin-top: 100px;
+  width: 60%;
+}
+
+.add-to-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.to-do {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  padding-left: 10px;
+  padding-right: 10px;
+  display: flex;
+  flex-direction: row;
+  text-align: left;
+}
+
+h1 {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-size: 25px;
+  font-weight: normal;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 10px;
+  padding-left: 0;
+  width: 100%;
+}
+
+#myInput {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  font-size: 20px;
+  font-style: italic;
+  border: none;
+  padding-left: 10px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.clear-btn {
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  border-radius: 0;
+  width: 100%;
+  font-size: 20px;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  border: none;
+  color: #808080;
```

Comment:
Consider creating a class with this color value ( `#808080` ) and add to all relevant elements. This color value is repeated a number of times. e.g: 
```
.my-color {
  color: #808080;
}